baybe/recommenders/base.py

```diff
@@ -43,6 +43,7 @@ def recommend(
         overrides=dict(
             allow_repeated_recommendations=cattrs.override(omit=True),
             allow_recommending_already_measured=cattrs.override(omit=True),
+            acquisition_function_cls=cattrs.override(omit=True),
```

Comment:
Out of curiosity, could you just quickly explain what this does?

Reply:
the old variable is just there to create an erorr when used and has no other function, as a result it should not be written to the serialized string. In fact if you were to serialize this variable you would auto-trigger the respective error you coded whenever the object is about to un-serialize

Reply:
Makes perfect sense, thanks for the explanation!